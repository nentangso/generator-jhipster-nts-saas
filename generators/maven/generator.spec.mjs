import { expect } from 'expect';

import { helpers, lookups } from '#test-utils';

const SUB_GENERATOR = 'maven';
const BLUEPRINT_NAMESPACE = `jhipster:${SUB_GENERATOR}`;

describe('SubGenerator maven of nts-saas JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create(BLUEPRINT_NAMESPACE)
        .withOptions({
          reproducible: true,
          defaults: true,
          blueprint: 'nts-saas',
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
