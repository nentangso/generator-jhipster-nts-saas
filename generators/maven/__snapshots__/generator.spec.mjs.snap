// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator maven of nts-saas JHipster blueprint run should succeed 1`] = `
Object {
  ".mvn/wrapper/maven-wrapper.jar": Object {
    "stateCleared": "modified",
  },
  ".mvn/wrapper/maven-wrapper.properties": Object {
    "stateCleared": "modified",
  },
  ".yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "mvnw": Object {
    "stateCleared": "modified",
  },
  "mvnw.cmd": Object {
    "stateCleared": "modified",
  },
  "pom.xml": Object {
    "stateCleared": "modified",
  },
}
`;
